assets/keymaps/atom.json

```diff
@@ -70,7 +70,6 @@
   {
     "context": "ProjectPanel",
     "bindings": {
-      "a": "project_panel::NewFile",
       "shift-a": "project_panel::NewDirectory",
```

Comment:
@notpeter I think we should solve this the way vim does instead:

```
"context": "Projectpanel && not_editing",
"bindings": {

...
}
```

Reply:
(https://github.com/zed-industries/zed/blob/449a278b8971673b650582d73c2ad008a609cae8/assets/keymaps/vim.json#L684)

Reply:
Perfect.  I realized 'A' and 'D' were also broken too and moved them into the same block